Week01/Problem05/apavlova_05.py

```diff
@@ -0,0 +1,19 @@
+"""Start with a number that is a product of all numbers from 2 and 20
+and gradually decrease it to fine the smallest one"""
+Prod=1
+for i in range(2,20):
+  Prod=Prod*i
+def CheckMult(x):
+   """Checks is the present product is still dividable by all numbers in 
+the list"""
```

Comment:
Yay, function docstrings!! These are totally sufficient for now, but in the future you may be interested in following some of the accepted docstring guidlines ([numpydoc](http://numpydoc.readthedocs.io/en/latest/format.html) is my personal choice).  

One good reason for doing this is that if you ever start contributing software to a big package, there are ways of automatically generating code documentation based on these docstrings, but they only know how to scrape ones which use standard formatting.